source/userguide/auto_annotator.rst

```diff
@@ -235,19 +260,72 @@ Some of these entities are resolvable by Duckling.
 +------------------------+-------------------------+-----------------------------------------------------------------------------+
 
-
-To detect entities in a single sentence first create an instance of the :class:`SpacyAnnotator` class.
+Supported languages include English (en), Spanish (es), French (fr), German (de), Danish (da), Greek (el), Portuguese (pt), Lithuanian (lt), Norwegian Bokmal (nb), Romanian (ro), Polish (pl), Italian (it), Japanese (ja), Chinese (zh), Dutch (nl).
+The table below identifies the supported entities for each language.
+
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+|                     | EN | ES | FR | DE | DA | EL | PT | LT | NB | RO | PL | IT | JA | ZH | NL |
++=====================+====+====+====+====+====+====+====+====+====+====+====+====+====+====+====+
+| sys_amount-of-money | y  | y  | y  | n  | n  | n  | y  | n  | y  | y  | n  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_distance        | y  | y  | y  | y  | n  | n  | y  | n  | n  | y  | n  | y  | n  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_duration        | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_event           | y  | n  | n  | n  | n  | y  | n  | n  | n  | y  | n  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_fac             | y  | n  | n  | n  | n  | n  | n  | n  | n  | y  | n  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_gpe             | y  | n  | n  | n  | n  | y  | n  | y  | n  | y  | y  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_interval        | y  | y  | y  | y  | y  | y  | y  | n  | y  | y  | y  | y  | n  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_language        | y  | n  | n  | n  | n  | n  | n  | n  | n  | y  | n  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_law             | y  | n  | n  | n  | n  | n  | n  | n  | n  | n  | n  | n  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_loc             | y  | y  | y  | y  | y  | y  | y  | y  | y  | y  | n  | y  | y  | y  | y  |
++---------------------+----+----+----+----+----+----+----+----+----+----+----+----+----+----+----+
+| sys_nrp             | y  | n  | n  | n  | n  | n  | n  | n  | n  | y  | n  | n  | y  | y  | y  |
```

Comment:
Noticed this being referred to as sys_norp in a couple of places.

Reply:
Left a comment in one of the code files. The other reference is in the previous table for "supported entities"

Reply:
Good catch, this should be "sys_norp" as to align with Spacy's labelling scheme.